### ISSUE TITLE
[Poco M3] random WhatsApp audio crackling

### ISSUE THREAD

**Issue by KaMyKaSii** [phhusson/treble_experimentations#1777]
In a few moments (i don't know what the trigger is) WhatsApp audios start popping a lot. Attached is a logcat with this happening.
[cracklingaudio.txt](https://github.com/phhusson/treble_experimentations/files/6167696/cracklingaudio.txt)
I've tried disabling audio effects but this hasn't fixed it. Is a systrace also required?

**Comment by phhusson** [comment]
Nope systrace is useful mostly for graphical smoothness (though sure it can be used for other stuff)

24 mars 2021 21:12:42 KaMyKaSii ***@***.***>:

**Comment by edwining01** [comment]
@KaMyKaSii Have you tried "Use alternate audio policy" in `Settings`?

**Comment by KaMyKaSii** [comment]
> @KaMyKaSii Have you tried "Use alternate audio policy" in `Settings`?

Yes, I enabled the alternative policy because in the default setting no headphones were identified. I will reinstall this GSI in the near future and see if anything has changed with the new updates

**Comment by edwining01** [comment]
I had this problem when using a new user to watch some videos. After enabled `Use alternate audio policy` for that user, no crackling sound found. If reinstall not work, here is an idea that it might need to flash and update one of the latest xiaomi rom first, especially the `/vendor` image. My device currently was flashed with the March EU rom before this GSI.

**Comment by KaMyKaSii** [comment]
> If reinstall not work, here is an idea that it might need to flash and update one of the latest xiaomi rom first, especially the `/vendor` image. My device currently was flashed with the March EU rom before this GSI.

Can you link it to me? And do you know if the vendor base is Android 10 or 11?

**Comment by edwining01** [comment]
https://androidfilehost.com/?fid=17248734326145681107
